src/ENCRYPTO_utils/sndthread.cpp

```diff
@@ -49,11 +50,17 @@ void SndThread::add_snd_task_start_len(uint8_t channelid, uint64_t sndbytes, uin
 	push_task(std::move(task));
 }
 
+void SndThread::add_snd_task_start_len(uint8_t channelid, uint64_t sndbytes, uint8_t* sndbuf, uint64_t startid, uint64_t len) {
+	//Call the method blocking but since callback is null nobody gets notified, other functionallity is equal
+	add_blocking_snd_task_start_len(NULL, channelid, sndbytes, sndbuf, startid, len);
```

Comment:
I think we should change NULL to nullptr here (and at some other places in this PR)